backend/metrics/calculations.py

```diff
@@ -107,6 +148,9 @@ def extend_and_impute_metrics(
             - df["delta.other"]
     )
 
+    df['total.confirmed.14_day_ratio'] = df.groupby(grouping_columns)["delta.hospitalized"].apply(
```

Comment:
I'll take a closer look at the implementation for this method, but what are the expected values for the "grouping_columns" list?

Reply:
**3 options**
state sheet: 'state'
district sheet: 'state', 'district'
ward sheet: 'state', 'district', 'ward'

depending on for which google sheet we are generating metrics, we have additional grouping columns.